etl/steps/data/garden/oecd/2023-12-06/pisa.py

```diff
@@ -0,0 +1,90 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("pisa")
+    # Read table from meadow dataset.
+    tb = ds_meadow["pisa_math_boys_girls"].reset_index()
+
+    #
+    # Process data.
+    #
+
+    tb = geo.harmonize_countries(
+        df=tb, excluded_countries_file=paths.excluded_countries_path, countries_file=paths.country_mapping_path
+    )
+
+    # Set index
+    tb = tb.set_index(["country", "year"], verify_integrity=True)
+
+    # Renaming columns to remove '_boys_girls'
+    tb = tb.rename(columns=lambda x: x.replace("_boys_girls", ""))
+
+    # Remove columns with standard errors for now
+    tb = tb.drop(
+        columns=[
+            "pisa_reading_se_girls",
+            "pisa_reading_se_boys",
+            "pisa_science_se_girls",
+            "pisa_science_se_boys",
+            "pisa_math_se_girls",
+            "pisa_math_se_boys",
+            "pisa_math_all_se",
+            "pisa_science_all_se",
+            "pisa_reading_all_se",
+        ]
+    )
+    # Add metadata.
```

Comment:
I wouldn't change what you've done here but I would recommend trying out the Jinja templates for metadata. 

It's essentially the same as what you've done but I find it somehow a little easier to work with. The main change is that you need wider tables, so a column for each disaggregation you would have different metadata for.